### PR TITLE
Added remove and back comnand

### PR DESCRIPTION
**Pull request by L0SER8228**
I've added remove comannd to remove a song from the queue. 
back command, to play back the previous song

### CODE REVIEW

**Comment by ZerioDev** [comment]
Good evening 👋

I look at this and if everything works i accept this.
Thank you and good evening !

File: commands/music/back.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    utilisation: '{prefix}back',
+
+    execute(client, message) {
+        if (!message.member.voice.channel) return message.channel.send(`${client.emotes.error} - You're not in a voice channel !`);
+
+        if (message.guild.me.voice.channel && message.member.voice.channel.id !== message.guild.me.voice.channel.id) return message.channel.send(`${client.emotes.error} - You are not in the same voice channel !`);
+
+        if (!client.player.getQueue(message)) return message.channel.send(`${client.emotes.error} - No music currently playing !`);
+
+        if (client.player.getQueue(message).previousTracks.length < 1) return message.channel.send(`${client.emotes.error}` - No previous track was found !);
```
Comment (Gravxd) [inline commands/music/back.js:14]: if (client.player.getQueue(message).previousTracks.length < 1) return message.channel.send(`${client.emotes.error} - No previous track was found !`);

**Review by Gravxd — COMMENTED** [review]
on line 14, there is a typo that would cause an issue with the response. ` needs to be moved to the right side of the response inside the brackets.

**Comment by L0SER8228** [comment]
> on line 14, there is a typo that would cause an issue with the response. ` needs to be moved to the right side of the response inside the brackets.

Thank you for review. I Fixed it 😊

**Comment by Gravxd** [comment]
> > on line 14, there is a typo that would cause an issue with the response. ` needs to be moved to the right side of the response inside the brackets.
> 
> Thank you for review. I Fixed it 😊

No Problem, i know its a basic typo but people will probs not notice it and complain it throws errors 😄

**Comment by revot334** [comment]
added in new update :) (not your code)

**Pull request closed without merging**